pycloudlib.toml.template

```diff
@@ -45,6 +45,15 @@ project = ""  # glcoud config get-value project
 # private_key_path = ""  # Defaults to 'public_key_path' without the '.pub'
 # key_name = ""  # Defaults to your username if not set
 
+[ibm]
```

Comment:
I think we might need to include VPC here.

E.g., if I run your cloud-init branch, I get `pycloudlib.ibm._util.IBMException: VPC not found: us-east-default-vpc` on every launch...even though I can see `us-east-default-vpc` if I log into the web console.

Reply:
Added the ability to set the VPC here.

The problem with that particular vpc (`us-east-default-vpc`) is that it is not associated to the resource group that you are probably using, otherwise it should have been detected.